lib/beaker/host_prebuilt_steps.rb

```diff
@@ -411,8 +411,10 @@ def enable_root_login host, opts
       block_on host do |host|
         logger.debug "Update /etc/ssh/sshd_config to allow root login"
         if host['platform'] =~ /osx/
-          host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin no/PermitRootLogin Yes/g' /etc/sshd_config"))
-          host.exec(Command.new("sudo sed -i '' 's/#PermitRootLogin yes/PermitRootLogin Yes/g' /etc/sshd_config"))
+          ssh_config_file = '/etc/sshd_config'
+          ssh_config_file = '/private/etc/ssh/sshd_config' if host['platform'] =~ /osx-10\.*(11|12)/
```

Comment:
To future-proof this a bit, we should just assume that `/private/etc/ssh/sshd_config` will be the location for future versions of osx. We should change this logic so that osx-1013, osx-1014, and so on will automatically get this new location, not the location that existed in osx-1010.